src/components/Create/Create.js

```diff
@@ -179,14 +258,65 @@ const Create = ({ history }) => {
                     required
                   />
                 </Form.Item>
-                <Form.Item>
-                  <Input
-                    placeholder="white paper/document hash"
-                    name="documentHash"
-                    onChange={e => handleChange(e)}
-                    required
-                  />
+                <Form.Item style={{ lineHeight: "0px" }}>
+                  <Dragger
+                    multiple={false}
+                    action="https://www.mocky.io/v2/5cc8019d300000980a055e76"
+                    transformFile={transformFile}
+                    beforeUpload={beforeUpload}
+                    onChange={handleChangeUpload}
+                    onRemove={() => false}
+                    fileList={fileList}
+                    name="documentHashUpload"
+                    style={{
+                      background: "#D3D3D3",
+                      borderRadius: "8px"
+                    }}
+                  >
+                    <Icon style={{ fontSize: "24px" }} type="upload" />
+                    <p>Click or drag file to this area to upload (optional)</p>
+                    <Input
+                      style={{ borderRadius: 0, align: "center" }}
+                      placeholder={"white paper/document hash (optional)"}
+                      name="documentHash"
+                      disabled
+                      value={hash}
+                    />
+                  </Dragger>
+                  {!loading && hash && (
+                    <>
+                      <p style={{ textAlign: "left", marginBottom: "-18px" }}>
+                        White paper/document hash:
+                      </p>
+                      <Paragraph small copyable={{ text: hash }} ellipsis>
+                        {hash}
+                      </Paragraph>
+                    </>
+                  )}
+
+                  <Collapse accordion>
```

Comment:
Please refactor so that the Document Hash option is entirely behind a Collapse div, the user should not see the `Click or drag file area` option unless clicking the Collapse div

Collapse div titled `Add white paper / document hash...`
On expand, show input field and the "What is a document hash?" section below